testing/trino-product-tests/src/main/java/io/trino/tests/product/exasol/TestExasol.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.exasol;
+
+import io.trino.tempto.ProductTest;
+import org.testng.annotations.Test;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.tempto.assertions.QueryAssert.assertThat;
+import static io.trino.tests.product.TestGroups.EXASOL;
+import static io.trino.tests.product.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+
+public class TestExasol
+        extends ProductTest
+{
+    private static final String SCHEMA_NAME = "test";
+    private static final String TABLE_NAME = SCHEMA_NAME + ".tab";
+
+    @Test(groups = {EXASOL, PROFILE_SPECIFIC_TESTS})
+    public void testSelect()
+    {
+        onTrino().executeQuery("CREATE SCHEMA " + SCHEMA_NAME);
+        try {
+            onTrino().executeQuery("create table " + TABLE_NAME + " (id integer, name varchar(5))");
+            assertThat(onTrino().executeQuery("SELECT COUNT(*) FROM " + TABLE_NAME))
```

Comment:
Can we insert rows on Exasol?

Reply:
No. I tried adding a method similar to `onMysql()` but it didn't work.

Reply:
What's the error message?

Reply:
I added method `onExasol()` with this line: `return testContext().getDependency(QueryExecutor.class, "exasol");`. This results in the following error:

```
tests               | 2023-03-08 19:04:14 SEVERE: Failure cause:
tests               | com.google.inject.ConfigurationException: Guice configuration errors:
tests               | 
tests               | 1) [Guice/MissingImplementation]: No implementation for QueryExecutor annotated with @Named("exasol") was bound.
tests               | 
tests               | Did you mean?
tests               |     * QueryExecutor annotated with @Named("hive")
tests               |     * QueryExecutor annotated with @Element(setName=,uniqueId=450, type=MAPBINDER, keyType=String)
tests               |     * QueryExecutor annotated with @Named("presto_no_default_catalog")
tests               |     27 more bindings with other annotations.
tests               | 
tests               | Learn more:
tests               |   https://github.com/google/guice/wiki/MISSING_IMPLEMENTATION
tests               | 
tests               | 1 error
tests               | 
tests               | ======================
tests               | Full classname legend:
tests               | ======================
tests               | Element:       "com.google.inject.internal.Element"
tests               | Named:         "com.google.inject.name.Named"
tests               | QueryExecutor: "io.trino.tempto.query.QueryExecutor"
tests               | ========================
tests               | End of classname legend:
tests               | ========================
tests               | 
tests               |   at com.google.inject.internal.InjectorImpl.getProvider(InjectorImpl.java:1127)
tests               |   at com.google.inject.internal.InjectorImpl.getInstance(InjectorImpl.java:1134)
tests               |   at io.trino.tempto.internal.context.GuiceTestContext.getDependency(GuiceTestContext.java:83)
tests               |   at io.trino.tempto.internal.context.GuiceTestContext.getDependency(GuiceTestContext.java:78)
tests               |   at io.trino.tests.product.utils.QueryExecutors.onExasol(QueryExecutors.java:88)
tests               |   at io.trino.tests.product.exasol.TestExasol.testSelect(TestExasol.java:37)
tests               |   at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
```
I am not sure how/where to add the `QueryExecutor` for Exasol.